ui/apps/platform/src/reducers/systemConfig.js

```diff
@@ -11,8 +12,20 @@ export const types = {
 
 // Actions
 
-export const actions = {
-    fetchPublicConfig: createFetchingActions(types.FETCH_PUBLIC_CONFIG),
+export const fetchPublicConfig = () => {
```

Comment:
A question, not necessarily a change request, is there a variation on `fetchPublicConfig` name?
* which removes the need to alias `fetchPublicConfig` service function
* which reduces possible confusion between the action and service functions for Find in Files (not a new problem: yesterday, because of same name for the pair of functions, I had to analyze each occurrence to confirm that the code no longer calls an action function, nor calls the service function directly, therefore the indirect call via sagas is obsolete).

Reply:
Thinking a bit more, without revealing too much about the implementation, this action function, and corresponding action functions for other forward-looking reducers, are only for `dispatch` call at top level, correct? So a naming convention might help communicate that they:
* are similar to each other
* are different from the other pure action functions in the reducer

Reply:
I mostly mimicked the pattern from other reducers but I agree. I updated multiple function names to prevent needing to alias